utils/components-testkit/src/main/scala/pl/touk/nussknacker/engine/util/test/TestScenarioRunner.scala

```diff
@@ -45,8 +47,12 @@ trait TestScenarioRunner
 
 trait TestScenarioRunnerBuilder[R <: TestScenarioRunner, B <: TestScenarioRunnerBuilder[R, _]] {
 
+  def componentUseCase(testRuntimeMode: Boolean): ComponentUseCase = if (testRuntimeMode) TestRuntime else EngineRuntime
```

Comment:
Let's move it to some utils. It's make no sense to put here method that takes some argument that can be not related with `testRuntimeMode` set by `inTestRuntimeMode` method. IMO we should transform this boolean to `ComponentUseCase` in the last possible moment